### PR TITLE
Add example events for datasets under aws package

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
This PR is to add example json events for all datasets under AWS package.
Also changed some aws to AWS.

### CODE REVIEW

File: dev/packages/alpha/aws/0.0.3/manifest.yml [hunk 1]
```diff
@@ -127,12 +127,12 @@ icons:
   type: image/svg+xml
 datasources:
 - name: aws
-  title: aws logs and metrics
-  description: Collect logs and metrics from aws instances
+  title: AWS logs and metrics
```
Comment (mtojek) [inline dev/packages/alpha/aws/0.0.3/manifest.yml:130]: It's good that fixed the spelling too. Could you please fix the package "title" as well? I think it's also lowercased..

File: dev/packages/alpha/aws/0.0.3/manifest.yml [hunk 2]
```diff
@@ -127,12 +127,12 @@ icons:
...
+  description: Collect logs and metrics from AWS instances
   inputs:
   - type: logs
-    title: Collect logs from aws instances
-    description: Collecting aws cloudtrail, cloudwatch, ec2, elb, s3access and vpcflow
+    title: Collect logs from AWS instances
+    description: Collecting AWS cloudtrail, cloudwatch, ec2, elb, s3access and vpcflow
```
Comment (mtojek) [inline dev/packages/alpha/aws/0.0.3/manifest.yml:135]: I think the spelling of AWS services can be also improved.

Comment (kaiyan-sheng) [reply dev/packages/alpha/aws/0.0.3/manifest.yml]: Done.

File: dev/packages/alpha/aws/0.0.3/manifest.yml (hunk 1 above)
Comment (kaiyan-sheng) [reply dev/packages/alpha/aws/0.0.3/manifest.yml:130]: Done

**Pull request merged**